Formula/v/vrc-get.rb

```diff
@@ -0,0 +1,25 @@
+class VrcGet < Formula
+  desc "Open Source alternative of Command-line client of VRChat Package Manager"
+  homepage "https://github.com/anatawa12/vrc-get"
+  url "https://github.com/anatawa12/vrc-get/archive/refs/tags/v1.7.0.tar.gz"
+  sha256 "585c3efec6ff57b03ad820da5387b63af26b54b0202d4e82dbb9d977c0d0aef3"
+  license "MIT"
+  head "https://github.com/anatawa12/vrc-get.git", branch: "master"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
```

Comment:
yeah, we can see how the default livecheck goes.

Reply:
I mean the default livecheck is already returning lots of unrelated versions:

```console
$ brew lc -d vrc-get
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FromDefaultNameLoader): loading vrc-get
/usr/local/Homebrew/Library/Homebrew/brew.rb (Cask::CaskLoader::NullLoader): loading vrc-get

Formula:          vrc-get
Livecheckable?:   No

URL:              https://github.com/anatawa12/vrc-get/archive/refs/tags/v1.7.0.tar.gz
URL (processed):  https://github.com/anatawa12/vrc-get.git
Strategy:         Git

Matched Versions:
0.1.0, 0.1.1, 0.1.2, 0.1.3, 0.1.10, 0.1.11, 0.1.12, 0.1.13, 0.1.4, 0.1.5, 0.1.6, 0.1.7, 0.1.8, 0.1.9, 0.2.0, 0.2.1, 0.2.2, 0.2.3, 0.2.4, 0.2.5, 0.2.6, 1.0.0, 1.0.1, 1.0.2, 1.1.0, 1.1.1, 1.1.2, 1.1.3, 1.2.0, 1.3.0, 1.3.1, 1.3.2, 1.4.0, 1.4.1, 1.4.2, 1.4.3, 1.5.0, 1.5.1, 1.5.2, 1.5.3, 1.6.0, 1.6.1, 1.7.0, 0.0.2, 0.0.3, 0.0.4, 0.0.5, 0.0.6, 0.0.7, 0.0.8, 0.0.9

vrc-get: 1.7.0 ==> 1.7.0
```

It may work currently but this is prune to errors.

Reply:
I have added livecheck. thank you for suggestion and sorry for late response